main.qml

```diff
@@ -924,13 +933,19 @@ ApplicationWindow {
                     + ", fee: " + walletManager.displayAmount(transaction.fee));
 
             // here we show confirmation popup;
+            var transactionAmount = Number(walletManager.displayAmount(transaction.amount));
```

Comment:
This results in small amounts getting displayed like this: `1e-8 XMR`, the average user doesn’t know what this means.